agent/agent.go

```diff
@@ -14,7 +14,9 @@ import (
 	"strings"
 	"sync"
 
+	bolt "github.com/coreos/bbolt"
```

Comment:
I just realized the asset manager uses `bolt "go.etcd.io/bbolt"`. Should we create an issue to migrate to only one of these repo?

Reply:
Oh, I meant to use that one. I'll change it for this PR.